Prose/ProseLib/Sources/SidebarFeature/SidebarView.swift

```diff
@@ -5,179 +5,80 @@
-            ),
-            toolbar: ToolbarState()
-        )
-    }
-}
-
-// MARK: Actions
-
-public enum SidebarAction: Equatable, BindableAction {
-    case content(SidebarContentAction)
-    case footer(FooterAction)
-    case toolbar(ToolbarAction)
-    case binding(BindingAction<SidebarState>)
-}
-
-// MARK: Environment
-
-public struct SidebarEnvironment {
-    let userStore: UserStore
-    let messageStore: MessageStore
-    let statusStore: StatusStore
-    let securityStore: SecurityStore
-
-    public init(
-        userStore: UserStore,
-        messageStore: MessageStore,
-        statusStore: StatusStore,
-        securityStore: SecurityStore
-    ) {
-        self.userStore = userStore
-        self.messageStore = messageStore
-        self.statusStore = statusStore
-        self.securityStore = securityStore
-    }
-}
-
-public extension SidebarEnvironment {
-    static var placeholder: SidebarEnvironment {
-        SidebarEnvironment(
-            userStore: .placeholder,
-            messageStore: .placeholder,
-            statusStore: .placeholder,
-            securityStore: .placeholder
-        )
+    private var spotlightSection: some View {
+        Section(L10n.Sidebar.Spotlight.title) {
+            IconRow(title: L10n.Sidebar.Spotlight.unreadStack, icon: .unread)
+                .tag(Tag.unreadStack)
+            IconRow(title: L10n.Sidebar.Spotlight.replies, icon: .reply)
+                .tag(Tag.replies)
+            IconRow(title: L10n.Sidebar.Spotlight.directMessages, icon: .directMessage)
+                .tag(Tag.directMessages)
+            IconRow(title: L10n.Sidebar.Spotlight.peopleAndGroups, icon: .group)
+                .tag(Tag.peopleAndGroups)
+        }
     }
-}
 
-public extension SidebarEnvironment {
-    static var stub: SidebarEnvironment {
-        SidebarEnvironment(
-            userStore: .stub,
-            messageStore: .stub,
-            statusStore: .stub,
-            securityStore: .stub
-        )
+    private var contactsSection: some View {
+        Section(L10n.Sidebar.TeamMembers.title) {
+            ContactRow(title: "John Doe", avatar: .placeholder).tag(Tag.chat("john@prose.org"))
+            ActionButton(title: L10n.Sidebar.TeamMembers.Add.label) {
+                self.viewStore.send(.addContactButtonTapped)
+            }
+        }
     }
-}
 
-public extension SidebarEnvironment {
-    var destination: NavigationDestinationEnvironment {
-        NavigationDestinationEnvironment(
-            userStore: self.userStore,
-            messageStore: self.messageStore,
-            statusStore: self.statusStore,
-            securityStore: self.securityStore
-        )
+    private var groupsSection: some View {
+        Section(L10n.Sidebar.Groups.title) {
+            IconRow(title: "My group", icon: .group)
+            ActionButton(title: L10n.Sidebar.Groups.Add.label) {
+                self.viewStore.send(.addGroupButtonTapped)
+            }
+        }
     }
```

Comment:
Are we hard-coding those names for now?

Reply:
Yes. Loading contacts from the Roster will be the next step. Multi-User-Chat and thus groups will still take some time.

Reply:
I meant why not hard-coding it as a default value in State, so we can override it, but it's useless, you're right